### ISSUE TITLE
Light Client Overhauls

### ISSUE THREAD

**Issue by Oschart** [Oschart/FLoBC#12]
The LC is expected to have the following capabilities:
1. Have its own training data
2. Be able to perform training by calling a Python script and extracting the resultant gradients
3. Always fetch the latest model before starting the training to be in sync 
4. Don't redo training if the latest model it trained on hasn't changed
